### PR TITLE
Impl try-runtime for Manta runtimes

### LINKED ISSUE #122: Scaffold `try-runtime` for Calamari/Manta
Split from https://github.com/Manta-Network/Manta/issues/110

- [x] Integrate into manta-pc
- [x] Integrate into calamari
- [x] Scaffold custom hooks in runtimes
- [ ] Scaffold hooks in pallet-manta-pay
- [ ] Assertions in the hooks for tests

https://www.crowdcast.io/e/substrate-seminar/41
1hour 15 minutes for test ideas.

### PR DESCRIPTION
**Pull request by ghzlatarev**
closes #122 

* initial ``try-runtime`` integration for ``manta-pc`` and ``calamari-pc``
* scaffold runtime hooks in ``manta-pc`` and ``calamari-pc``
* update CI to check ``--all-features``

* you can build the binaries with ``cargo build --release --features=manta-pc,try-runtime``
* you can explore the command with ``./target/release/manta-pc  try-runtime --help``

### CODE REVIEW

File: node/Cargo.toml [hunk 1]
```diff
@@ -99,7 +100,15 @@ manta-primitives = { path = '../runtime/primitives' }
...
-default = []
+default = ["cli"]
+cli = [
+	"try-runtime-cli",
+]
+try-runtime = [ 
+	"manta-pc-runtime/try-runtime",
+	"calamari-runtime/try-runtime",
+]
+
```
Comment (ghzlatarev) [inline node/Cargo.toml:111]: Not quite sure what this is for. Check the error again when it is commented out.

File: node/Cargo.toml [hunk 2]
```diff
@@ -39,6 +39,7 @@ jsonrpc-core = "15.1.0"
 # Substrate frames
 frame-benchmarking = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
 frame-benchmarking-cli = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
+try-runtime-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", optional = true }
```
Comment (xialovexia) [inline node/Cargo.toml]: may need to specify a version here.

Comment (ghzlatarev) [reply node/Cargo.toml]: You got it. Included the version explicitly it is 0.9.0, but don't get it confused with the polkadot v0.9.8

**Review by Dengjianping — APPROVED** [review]
LGTM!
Good work!

**Pull request merged**